app/helpers/application_helper.rb

```diff
@@ -641,5 +641,10 @@ def list_to_string(list)
     list&.join(',') || ''
   end
 
+  def render_rounded_button_with_style(style, link)
```

Comment:
I think you have already created a function like this in the past, please check to avoid repetition.